content/source/docs/enterprise/before-installing/network-requirements.html.md

```diff
@@ -11,7 +11,7 @@ The Linux instance that runs Terraform Enterprise needs to allow several kinds o
 
 ### Source - User/Client/VCS
 
-* **443**: To access the Terraform Enterprise application via HTTPS 
+* **443**: To access the Terraform Enterprise application via HTTPS
```

Comment:
```suggestion
* **443**: To access the Terraform Enterprise application via HTTPS 
```

Normally I'd leave this alone, but there's another PR or two that touch this text, so we can avoid merge conflicts by leaving the unwanted space in place.